hw_4/task_3.js

```diff
@@ -0,0 +1,40 @@
+function word(wordSet) {
+    let word = wordSet.toLocaleLowerCase()
+vowels = "aeiou"
+vowels_count = 0
+consonants_count = 0
+for(const i of word){
+    if(vowels.includes(i)) vowels_count++; 
+	else if (typeof i === "symbol") {
+		consonants_count++;
+	}
+}
+console.log(`word contains vowels: ${vowels_count}, and  consonants: ${consonants_count}`);
+}
+
+word("ab21")
```

Comment:
With this solution I got such result - `word contains vowels: 1, and  consonants: 0`
but desired result is `word contains vowels: 1, and  consonants: 1`